core/src/components/modal/test/sheet/modal.e2e.ts

```diff
@@ -0,0 +1,136 @@
+import { expect } from '@playwright/test';
+import { test, dragElementBy } from '@utils/test/playwright';
+
+test.describe('sheet modal: rendering', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto('/src/components/modal/test/sheet');
+    const ionModalDidPresent = await page.spyOnEvent('ionModalDidPresent');
+
+    await page.click('#sheet-modal');
+
+    await ionModalDidPresent.next();
+
+    expect(await page.screenshot()).toMatchSnapshot(`modal-sheet-present-${page.getSnapshotSettings()}.png`);
+  });
+});
+
+test.describe('sheet modal: backdrop', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/src/components/modal/test/sheet');
+  });
+  test('should dismiss the sheet modal when clicking the active backdrop', async ({ page }) => {
+    const ionModalDidPresent = await page.spyOnEvent('ionModalDidPresent');
+    const ionModalDidDismiss = await page.spyOnEvent('ionModalDidDismiss');
+
+    await page.click('#backdrop-active');
+
+    await ionModalDidPresent.next();
+
+    await page.mouse.click(50, 50);
+
+    await ionModalDidDismiss.next();
+  });
+  test('should present another the sheet modal when clicking an inactive backdrop', async ({ page }) => {
+    const ionModalDidPresent = await page.spyOnEvent('ionModalDidPresent');
+
+    await page.click('#backdrop-inactive');
+    await ionModalDidPresent.next();
+
+    await page.click('#custom-height-modal');
+    await ionModalDidPresent.next();
+
+    await page.waitForSelector('.custom-height', { state: 'visible' });
+  });
+  test('input should outside sheet modal should be focusable when backdrop is inactive', async ({ page }) => {
```

Comment:
Looks like the test `'input should not be focusable when backdrop is active'` was skipped, was that intentional?

Reply:
Just added the test -good catch!